samples/nrf9160/nrf_cloud_mqtt_multi_service/src/fota_support.h

```diff
@@ -8,25 +8,12 @@
 
 #include <zephyr/kernel.h>
 
-/**
- * @brief Check whether we are capable of Firmware Over The Air (FOTA) application or modem update.
- *
- * @return bool - Whether we are capable of application or modem FOTA.
- */
-static inline bool fota_capable(void)
-{
-	return IS_ENABLED(CONFIG_NRF_CLOUD_FOTA) && IS_ENABLED(CONFIG_BOOTLOADER_MCUBOOT);
-}
-
-/**
- * @brief Check whether we are capable of Firmware Over The Air (FOTA) bootloader update.
- *
- * @return bool - Whether we are capable of bootloader FOTA.
+#if defined(CONFIG_NRF_CLOUD_FOTA_FULL_MODEM_UPDATE)
```

Comment:
I don't think this needs to be wrapped; But I do think `EXT_FLASH_DEVICE` could stand to be named `FULL_MODEM_FOTA_FLASH_DEVICE`